images/cilium/download-hubble.sh

```diff
@@ -17,7 +17,6 @@ for arch in amd64 arm64 ; do
   printf "%s %s" "${hubble_sha256[${arch}]}" "/tmp/hubble-${arch}.tgz" | sha256sum -c
   mkdir -p "/out/linux/${arch}/bin"
   tar -C "/out/linux/${arch}/bin" -xf "/tmp/hubble-${arch}.tgz" hubble
+  [[ "${arch}" = "amd64" ]] && x86_64-linux-gnu-strip /out/linux/amd64/bin/hubble
+  [[ "${arch}" = "arm64" ]] && aarch64-linux-gnu-strip /out/linux/arm64/bin/hubble
```

Comment:
Also, this doesn't necessary needs to be in the for loop, it can be after, same for the other ones.

Reply:
Yes, they are of the same function.
On the other side, we could think in this way: after we got 1 binary (hubble here), we do the strip operation then. 
So I think both of them are ok.